getting-started/introducing-tal.md

```diff
@@ -36,16 +36,26 @@ that it can be deployed to all HTML-based TV devices.
   </li>
 </ul>
 
-*iPlayer*, *BBC News*, and *BBC Sport* are three high profile TV applications built on TAL. Click on the images for larger versions:
+*iPlayer*, *Red Button+*, *BBC News*, and *BBC Sport* are four high profile TV applications built on TAL. Click on the images for larger versions:
 
 <ul class="thumbnails">
   <li class="span8 offset2">
     <div class="thumbnail">
-      <a href="{{site.baseurl}}/img/getting-started/iPlayer.jpg"><img src="{{site.baseurl}}/img/getting-started/iPlayer_50.jpg" alt=""></a>
+      <a href="{{site.baseurl}}/img/getting-started/iPlayer.jpg"><img src="{{site.baseurl}}/img/getting-started/iPlayer_new.jpg" alt=""></a>
```

Comment:
This link is broken - it links to the old iPlayer image.

This probably means you want a large new iPlayer screenshot to link to.